libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/parentFilters/ParentFiltersListItem.tsx

```diff
@@ -56,17 +55,14 @@ export const ParentFiltersListItem: React.FC<IConfigurationParentItemProps> = (p
     }, [isSelected, title]);
 
     const onSelect = useCallback(() => {
-        onClick(
-            localIdentifier,
-            !isSelected,
-            connectingAttributes.map((attr) => attr.ref),
-        );
+        const connectingAttributeRefs = connectingAttributes.map((attr) => attr.ref);
+        onClick(localIdentifier, !isSelected, connectingAttributeRefs);
     }, [isSelected, localIdentifier, onClick, connectingAttributes]);
 
     if (isDisabled) {
         return (
             <ParentFiltersDisabledItem
-                hasConnectingAttributes={!!connectingAttributes.length}
+                hasConnectingAttributes={true}
```

Comment:
Is this intentional? The previous `!!connectingAttributes.length` seemed reasonable.

Reply:
forgotten workaround for implementation steps. Moved back to length of connecting attributes